cfme/services/catalogs/catalog_items/__init__.py

```diff
@@ -826,3 +827,26 @@ class EditTags(CFMENavigateStep):
 
     def step(self, *args, **kwargs):
         self.prerequisite_view.policy.item_select('Edit Tags')
+
+
+class SetOwnershipView(ServicesCatalogView):
```

Comment:
It would be nice if you could move this view with other views in the top part of this file.

Reply:
Done

Reply:
Done